server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -916,29 +951,34 @@ public VolumeVO resizeVolume(ResizeVolumeCmd cmd) throws ResourceAllocationExcep
                 // no parameter provided; just use the original size of the volume
                 newSize = volume.getSize();
             }
-
+            newSizeInGb = newSize >> 30;
```

Comment:
Let's not use bit shift here. It makes things harder to read and understand.

The code is already convoluted enough